src/babashka/pods/sci.clj

```diff
@@ -19,6 +24,40 @@
               (string? var-value)
               (sci/eval-string* ctx var-value))))))
 
+(defn metadata-cache-file [pod-spec {:keys [:version]}]
+  (io/file (resolver/cache-dir {:pod/name pod-spec :pod/version version})
+           "metadata.cache"))
+
+(defn load-metadata-from-cache [pod-spec opts]
+  (let [cache-file (metadata-cache-file pod-spec opts)]
+    (when (.exists cache-file)
```

Comment:
This likely results into a reflection issue. Let's add *warn-on-reflection* in this namespace.